### PR TITLE
Fix session api update and add multiple organization support

### PR DESCRIPTION
**Pull request by CognitiveDisson**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -46,13 +46,44 @@ Or install it yourself as:
...
+  fabricio app all
+
+  # Obtain single app
+  fabricio app get --app_id 'app_id'
+
+  # Obtain active now count
+  fabricio app active_now --org_id 'org_id' --app_id 'app_id'
+
+  # Obtain single issue
+  fabricio app single_issue --app_id 'app_id' 'issue_id'
```
Comment (etolstoy) [inline README.md:60]: Did you forget the flag `--issue_id`?

File: README.md [hunk 2]
```diff
@@ -46,13 +46,44 @@ Or install it yourself as:
...
+
+  # Obtain single issue session
+  fabricio app issue_session  --app_id 'app_id' 'issue_id' 'session_id'
```
Comment (etolstoy) [inline README.md:63]: The same question about flags

File: README.md [hunk 3]
```diff
@@ -46,13 +46,44 @@ Or install it yourself as:
...
+
+  # Obtain latest issue session
+  fabricio app latest_session --app_id 'app_id'
+
+  # Obtain all builds
```
Comment (etolstoy) [inline README.md]: Let's divide the documentation into sections:

- App
- Build
etc

File: README.md [hunk 4]
```diff
@@ -46,13 +46,44 @@ Or install it yourself as:
...
+  fabricio build all --org_id 'org_id' --app_id 'app_id'
+
+  # Obtain single build
+  fabricio build get --org_id 'org_id' --app_id 'app_id' 'version' 'build_number'
```
Comment (etolstoy) [inline README.md:74]: You forgot some flags

File: lib/fabricio/cli/cli_helper.rb [hunk 5]
```diff
@@ -1,46 +1,34 @@
...
-FABRIC_GRAPHQL_API_URL = 'https://api-dash.fabric.io/graphql'
-
 def client
-  email = ""
-  password = ""
-  if File.file?(CREDENTIAL_FILE_PATH)
-    credential = YAML.load_file(CREDENTIAL_FILE_PATH)
-    email = credential['email']
-    password = credential['password']
+  sessionStorage = Fabricio::Authorization::FileSessionStorage.new()
```
Comment (etolstoy) [inline lib/fabricio/cli/cli_helper.rb]: We don't use camel case

File: lib/fabricio/services/app_service.rb [hunk 6]
```diff
@@ -35,32 +33,35 @@ def all
...
-      # @param id [String] Application identifier
       # @return [Fabricio::Model::App]
-      def get(id)
-        request_model = @request_model_factory.get_app_request_model(id)
+      def get(app_id = nil)
+        request_model = @request_model_factory.get_app_request_model(app_id)
         response = @network_client.perform_request(request_model)
-        Fabricio::Model::App.new(JSON.parse(response.body))
+        json = JSON.parse(response.body)
+        # TODO Handle error
```
Comment (etolstoy) [inline lib/fabricio/services/app_service.rb]: ToDo? :)

File: spec/lib/fabricio/service/app_service_get_stub_response.txt [hunk 7]
```diff
@@ -20,7 +20,7 @@
     "organization_id": "org_id",
     "watched": null,
     "importance_level": null,
-    "dashboard_url": "https://www.fabric.io/ramblerco/ios/apps/ru.ramblerco.livejournal",
+    "dashboard_url": "https://www.fabric.io/ramblerco/ios/apps/ru.organization.app",
```
Comment (etolstoy) [inline spec/lib/fabricio/service/app_service_get_stub_response.txt]: Replace `ramblerco` in these places also

**Review by etolstoy — APPROVED** [review]
Great pull request! Thanks @CognitiveDisson.

File: README.md (hunk 1 above)
Comment (CognitiveDisson) [reply README.md:60]: No.  It's a required parameter, alias is not needed.

File: lib/fabricio/services/app_service.rb (hunk 6 above)
Comment (CognitiveDisson) [reply lib/fabricio/services/app_service.rb]: I will create a task for this. This is a very important functionality

**Pull request merged**